### DISCUSSION TITLE
Responsive table with flex

### DISCUSSION THREAD

**Discussion by albertcito** [tailwindlabs/tailwindcss#12657] [Help] [answered]
I have a [tailwind code here (113)](https://play.tailwindcss.com/Vc1YqLGP07) where there is a sidebar menu  and a table in the right. I added `overflow-x-auto` to wrap the table, but in any case the scroll is not display in small screen, only the browser school but not the table div scroll. Someone know how to display the table scroll instead the window scroll?

```html
<div class="relative overflow-x-auto">
   <table class="w-full table-auto font-light">
```

The code is like
```html
<div class="flex ">
   <aside> ...</aside>
   <div class="flex-1">
     <header>  ...  </header>
     <div class="relative overflow-x-auto">
         <table class="w-full table-auto font-light">
              ...
         </table>
     </div>
   </div>
</div>
```

**Comment by wongjn** [comment, marked as answer]
Apply `min-width: 0` to override the implicit `min-width: min-content` applied to the `div.flex-1` element: https://play.tailwindcss.com/o5mh2OMAcL
